### PR TITLE
Fix the observer diag filenames to be ingested in EnKF

### PR DESCRIPTION
**Pull request by chunhuazhou**
## DESCRIPTION OF CHANGES: 
Following recent change to GSI diag filenames (change from *.nc4 to *.nc4.gz), fixing EnKF run script and workflow to adapt to the change.

## TESTS CONDUCTED: 
Tested on Hera and Jet

### Machines/Platforms:

- WCOSS2
  - [ ] Cactus/Dogwood
  - [ ] Acorn
- RDHPCS
  - [ x] Hera
  - [ x] Jet
  - [ ] Orion
  - [ ] Hercules

### Test cases: 

- [ ] Engineering tests
  - [ ] Non-DA engineering test
  - [ ] DA engineering test
    - [x ] Retro
    - [ x] Ensemble
    - [ ] Parallel
- [ ] RRFS fire weather
- [x ] RRFS_A:
- [ ] RRFS_B:
- [ ] RTMA:
- [ ] Others:

## ISSUE: 
https://github.com/NOAA-EMC/rrfs-workflow/issues/207

## CONTRIBUTORS (optional): 
@hu5970

### CODE REVIEW

**Review by hu5970 — APPROVED** [review]

File: scripts/exrrfs_run_enkf.sh
Comment (MatthewPyle-NOAA) [inline scripts/exrrfs_run_enkf.sh:1]: Could you change the mv_vrfy to mv in line 202?  We've moved away from the _vrfy commands as we prepare this workflow for operations.

**Review by MatthewPyle-NOAA — COMMENTED** [review]
One small item to change, then will approve.

File: scripts/exrrfs_run_enkf.sh
Comment (chunhuazhou) [reply scripts/exrrfs_run_enkf.sh:1]: Done. Thanks!

**Review by MatthewPyle-NOAA — APPROVED** [review]
Thanks for the quick fix - approving.

**Pull request merged**